### ISSUE TITLE
Almost everything is broken in chrome os 106

### ISSUE THREAD

**Issue by cdd100** [dnschneid/crouton#4830]
It isn't so much a single chroot, but whenever i try to get a desktop environment, it fails with a dpkg error. When I use tasksel, i get an apt-get error. It will not create a new chroot or update. I have a lenovo c330.

**Comment by TomTravis** [comment]
you install device location need to be exce and you must have exec  priv

chromeOS now tracks UID and privs the kills apt-get  writes to the device 
mount the devices exec every time it boots (start or restart)

**Comment by TomTravis** [comment]
![Screenshot at 2022-10-25 14:22:51](https://user-images.githubusercontent.com/46386261/197885162-971345ec-4429-4168-ab6b-af563b37daaa.png)
works for me

**Comment by dnschneid** [comment]
@cdd100 there's a number of known issues right now (they're pinned), but if you paste some logs from your install/run attempts, it'll be easier for people to help

**Comment by TomTravis** [comment]
it not 106 my installs work correct something was updated on this account rep Crouton
my fork runs no issue

**Comment by Pat-Anderson** [comment]
> you install device location need to be exec and you must have exec privs
> 
> chromeOS now tracks UID and privs this kills apt-get writes to the device mount the devices exec every time it boots (start or restart)
Could you please explain in plain English step by step how to do this for us mere mortals who are not rocket scientists? Would be much appreciated
